group_vars/all.yml

```diff
@@ -2,25 +2,31 @@
 # maintains ports, bianry or repo urls
 
 org: encapsulate
+domain_regex: ^([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}$
+monitor_server_dns: monitor.encapsulate.xyz
+ansible_port: 8192
+
+architecture_map:
+  x86_64: amd64
+  aarch64: arm64
 
 default_node_types:
   - validator
   - default
 
-sui:
-  validator:
-    source_url: https://releases.sui.io
-    ports:
-      server_listen_port: 9191
-      metrics_port: 9186
-  fullnode:
-    version: 88888888888
-    source_url: https://releases.sui.io
-    ports:
-      server_listen_port: 9191
-      metrics_port: 9186
+tmp_dir: /tmp
+
+go:
+  root: /usr/local/go
+  path: "{{ tmp_dir }}/go"
+  module111: "on"
+
+celestia:
   bridge:
-    source_url: https://releases.sui.io
+    source_url: https://github.com/celestiaorg/celestia-node
     ports:
-      server_listen_port: 9191
-      metrics_port: 9186
+      prefix: 104
```

Comment:
we wont really require prefix here, since this will be running on its own machine, lets use the default prefix, and keep the structure same